src/Tizen.NUI/src/public/BaseComponents/TextConstants.cs

```diff
@@ -546,6 +547,74 @@ public struct TextFit : IEquatable<TextFit>
         public override int GetHashCode() => (Enable, MinSize, MaxSize, StepSize, FontSizeType, FontSize).GetHashCode();
     }
 
+    /// <summary>
+    /// A struct to pass data of TextFitArray's OptionList.
+    /// </summary>
+    /// <remarks>
+    /// The TextFitArrayOption struct is used as an item to TextFitArray's OptionList. <br />
+    /// See <see cref="Tizen.NUI.Text.TextFitArray"/>.
+    /// </remarks>
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1815: Override equals and operator equals on value types")]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public struct TextFitArrayOption
+    {
+        /// <summary>
+        /// Constructor.
+        /// </summary>
```

Comment:
Shouldn't we add the description for the parameter?
```suggestion
        /// </summary>
        /// <param name="pointSize">The PointSize for TextFitArrayOption</param>
```

Reply:
I added description for the parameter, thanks

Reply:
Fast update :+1: